### PR TITLE
logitech-bulkcontroller: Speed up enumeration

### PR DESCRIPTION
**Pull request by hughsie**
We've worked out the state machine so we're not drowning in retry loops.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by vcdmp** [comment]
Wow. Looking good, Reviewing, also let me run full path. Is it going to ignore N/A events/messages?
 
In other words, if we send LOGI__DEVICE__PROTO__REQUEST__PAYLOAD_GET_DEVICE_INFO_REQUEST, we wait for LOGI__DEVICE__PROTO__RESPONSE__PAYLOAD_GET_DEVICE_INFO_RESPONSE, and ignore other messages/events like LOGI__DEVICE__PROTO__EVENT__PAYLOAD_CRASH_DUMP_AVAILABLE_EVENT.

I still haven't heard back from our offshore team for updated state-machine docs. Also reaching out the folks for git access.

**Comment by vcdmp** [comment]
Encountering intermittent failures, investigating. Logs attached. file:///home/ubuntu22/Firmware/Kong/fwupdtool-get-devices-output.zip

**Comment by hughsie** [comment]
> Is it going to ignore N/A events/messages?

In some cases, not. I've added a fixup to retry the command. Can you try again and then *attach* the logs please.

File: plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller.rs [hunk 1]
```diff
@@ -42,12 +42,18 @@ enum LogitechBulkcontrollerCmd {
...
     cmd: LogitechBulkcontrollerCmd,
     payload_length: u32le,
     sequence_id: u32le,
 }
+#[derive(Parse)]
+struct LogitechBulkcontrollerSendSyncRes {
+    cmd: LogitechBulkcontrollerCmd,
+    payload_length: u32le,
+    sequence_id: u32le,
+}
```
Comment (vcdmp) [inline plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller.rs:57]: Not familiar with .rs file syntax. Is "_" intended here?
    _payload_length: u32le,

Comment (hughsie) [reply plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller.rs:57]: Yup, that just means "reserved" -- i.e. we don't care.

File: plugins/logitech-bulkcontroller/README.md [hunk 2]
```diff
@@ -27,13 +27,59 @@ These devices use the standard USB DeviceInstanceId values, e.g.
 
 This plugin uses the following plugin-specific quirks:
 
+### `Flags=check-buffer-size`
```
Comment (vcdmp) [inline plugins/logitech-bulkcontroller/README.md:30]: Please include quirk file, if modified

**Review by vcdmp — APPROVED** [review]

File: plugins/logitech-bulkcontroller/README.md (hunk 2 above)
Comment (hughsie) [reply plugins/logitech-bulkcontroller/README.md:30]: I've left this disabled for now, until we work out the 16kb bug-- we can add it per-device when we know what devices and what versions work.

**Pull request merged**